src/analysis/seminorm.lean

```diff
@@ -821,11 +867,16 @@ end seminorm
 /-! ### The norm as a seminorm -/
 
 section norm_seminorm
-variables (𝕜 E) [normed_field 𝕜] [semi_normed_group E] [normed_space 𝕜 E] {r : ℝ}
+variables (𝕜) {E} [normed_field 𝕜] [semi_normed_group E] [normed_space 𝕜 E] {r : ℝ}
+
+lemma norm_eq_of_add_eq_zero {x y : E} (hxy : x + y = 0) : ∥ x ∥ = ∥ y ∥ :=
```

Comment:
Why not just make this say `∥-x∥ = ∥x∥` for any `x`?

Reply:
That's `norm_neg`. I wrote this to apply it directly in the definition `norm_add_monoid_seminorm`, but I could just inline the proof there.

Reply:
Oh I see; yes I would inline this.